modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/SpringCodegen.java

```diff
@@ -590,6 +598,16 @@ public void processOpts() {
         }
     }
 
+    private boolean containsEnums() {
+        if (this.openAPI == null) {
+            return false;
+        } else {
```

Comment:
Else is not necessary. You can remove it :)

Reply:
Ok